client/src/components/Page/AllRunsPage.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+import AppHeader from "../../components/AppHeader/AppHeader";
+import BottomNav from "../../components/BottomNav/BottomNav";
+import RunCard from "../../components/RunCard/RunCard";
+
+const Header = styled.div``;
+
+const Card = styled.div``;
+
+const Navigation = styled.div``;
```

Comment:
Not all of these components should be `div`. 
And I would only add styled-components, if you have stylings :).

Reply:
You added these lines in multiple pages. 
Try to only add code which are required for your project and avoid redundant code.